src/components/override-sdk/infra/Assignment/Assignment.tsx

```diff
@@ -580,7 +584,7 @@ export default function Assignment(props) {
           )}
 
           {arSecondaryButtons?.findIndex(button => button.name === 'Previous') === -1 &&
-          isCHBJourney() &&
+          (isCHBJourney() || isEduStartJourney()) &&
```

Comment:
Is this going to mean for every journey we are going to have to do a check?

Reply:
yes